CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.0](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.0) (2018-11-23)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/v4.0.0...v4.1.0)
+
+**Implemented enhancements:**
+
+- Implement snmpv3\_user fact and snmp::snmpv3\_usm\_hash function [\#157](https://github.com/voxpupuli/puppet-snmp/pull/157) ([smoeding](https://github.com/smoeding))
+
+**Closed issues:**
+
+- Move the module to voxpupuli [\#84](https://github.com/voxpupuli/puppet-snmp/issues/84)
+
+**Merged pull requests:**
+
+- Install snmptrapd on latests ubuntu [\#168](https://github.com/voxpupuli/puppet-snmp/pull/168) ([amateo](https://github.com/amateo))
```

Comment:
From the commit it looks like Debian was also added. I'd modify the PR title and regenerate the changelog. Given they're listed as supported distros in metadata.json it's IMHO a bugfix as well.

Reply:
For sure it is a bugfix. I added tag `bug`